gis/bylaw_text_to_centreline/README.md

```diff
@@ -0,0 +1,300 @@
+
```

Comment:
This folder should be `text_to_centreline`
The posted speed limit folder should be under this folder structure